draft-iab-nemops-workshop-report.md

```diff
@@ -567,25 +573,27 @@ This document does not have any IANA considerations.
 
 --- back
 
-# Insights from Operator Feedback {#insights}
+# Operator Feedback {#insights}
+
+This section compiles the operator feedback gathered through outreach and information gathering at various operational venues (RIPE, NANOG, APRICOT, LACNIC, AutoConn, etc). The PC synthesized this input and presented it during the workshop (see {{survey}}).
 
-## General Insights
+## General
 
 1. In network deployments, operations are typically at the bottom of the ladder. It's the most squeezed for time and resources. Network engineers are not typically seasoned developers. The development of needed in-house tools often takes years to develop. There is a need for tools that are easy to use and just work.
 2. The vast majority of smaller operators use CLI and open source to manage their networks.
 3. There is debate fatigue. The protocol/model debate is a recurring conversation. The problem isn’t going away.
 4. It was suggested that other domains (e.g., K8N/automation) are years ahead of the current network engineering stack.
 5. Support for multiple friendly, stable and feature-rich libraries for programming languages is needed. Many DevOps routines use shell scripts, while others use a high-level programming language. In any case, on the client side, multiple programming languages are used.
-6. Screen scraping is both necessary and evil. This most often occurs when interacting with a device having only a CLI.
+6. Screen scraping is both necessary and painful. This most often occurs when interacting with a device having only a CLI.
```

Comment:
So I think we don't want either evil or painful.  The point behind "evil" was that it shouldn't be needed.  So painful doesn't quite get that aspect, though is probably a better word choice.  How about "unfortunately necessary and painful" or "painful and unfortunately necessary because standard APIs do not exist" or ...

Reply:
will change to "unfortunately necessary and painful"